_playwright-tests/UI/helpers/deletePopularRepositories.ts

```diff
@@ -29,7 +29,5 @@ export const deleteAllPopularRepos = async ({ request }: Page) => {
       console.error('Failed to delete repositories:', error);
       throw error;
     }
-  } else {
-    console.log('No repositories to delete.');
```

Comment:
Why is this deleted?

Reply:
because it unnecessarily spams the PW test report and given that this function its used for auto cleanup this log isn't useful 😅